testing/ios/IosUnitTests/run_tests.sh

```diff
@@ -12,6 +12,6 @@ fi
 
 set -o pipefail && xcodebuild -sdk iphonesimulator \
   -scheme IosUnitTests \
-  -destination 'platform=iOS Simulator,name=iPhone SE,OS=12.2' \
+  -destination 'platform=iOS Simulator,name=iPhone 8,OS=13.0' \
```

Comment:
This simulator variant won't be available on ToT.

Reply:
Reverted the OS version, but kept it on iPhone 8 because my local environment doesn't even offer an iPhone SE

Reply:
FWIW you should be able to add iPhone SE locally, but AFAIK know of our tests depend on the simulator type at this point.